assets/javascripts/discourse/components/admin/filter-children-form.hbs

```diff
@@ -0,0 +1,52 @@
+{{#if (eq @toggleChildMenuFor @filterTag.name)}}
+  {{#unless @editingChildFor}}
+    <div class="child-picker-spacer">
+      <label class="global-filter-label">
+        {{i18n "global_filter.admin.filter_child_name"}}
+      </label>
+      <TagChooser
+        @tags={{@filterChildName}}
+        @onChange={{action @setFilterChildName}}
+        @options={{hash limit=1}}
+      />
+    </div>
+  {{/unless}}
```

Comment:
Don't allow the tag selected to be the filter child to be edited. It should be deleted and a new filter child created in its place.